redash/cli/__init__.py

```diff
@@ -48,7 +51,7 @@ def status():
 @manager.command()
 def check_settings():
     """Show the settings as Redash sees them (useful for debugging)."""
-    for name, item in settings.all_settings().iteritems():
+    for name, item in current_app.config.iteritems():
```

Comment:
Won't this show settings only Flask recognizes?

Reply:
Nope, it'll do a [`dir(redash.settings)`](https://github.com/pallets/flask/blob/1c2a2cd0014693f1e1f894e498c2dc50dbb44f66/flask/config.py#L139-L171) and check if the variables name is all upper case. We'd lose the check for `__` at the start that Redash's code does, but I couldn't find any settings that uses those.